CMake/kwiver-depends-GDAL.cmake

```diff
@@ -0,0 +1,11 @@
+# Optionally find and configure GDAL dependency
+
+option( KWIVER_ENABLE_GDAL
+  "Enable GDAL dependent code and plugins (Arrows)"
+  ${fletch_ENABLE_GDAL}
+  )
+
+if( KWIVER_ENABLE_GDAL )
+  find_package( GDAL REQUIRED )
```

Comment:
It might be important to add a version check to this find_package. You should prefer/require GDAL 2 over GDAL 1. The two are different enough to prevent trying to support both.

Reply:
I followed the example from vidtk to check the version number. It currently expects 1.11.5 since that is the default in fletch and 2.2.2 fails to build for me due to issues with geotiff.

Reply:
I didn't see the code you added, but we should not use GDAL 1.xx, only 2.xx unless the code your writing works in both. I just assumed because of the GDAL API changes that it wouldn't. FWIW, I don't really like what's going on in vidtk because it expects an exact match which is cumbersome when we do minor GDAL version updates. Probably the correct thing to do is have a small try/compile which ensures your code works against the found GDAL.

Reply:
@dstoup While I agree that we should move forward with GDAL 2.xx, the default in Fletch is still 1.xx.  Should we update Fletch to make GDAL 2 the default?  What is holding that back?

Reply:
When I switch GDAL from 1.11.5 to 2.2.2 I get build errors which I believe are related to libgeotiff. I see this on both MacOS and Linux.

Reply:
For the record, I think the consensus is that @dstoup will update Fletch to build GDAL 2.x by default and help fix any geotiff related issues that causes.  Then this branch should be updated to use GDAL 2.x